examples/Console/Program.cs

```diff
@@ -27,18 +27,21 @@ public class Program
         /// Main method - invoke this using command line.
         /// For example:
         ///
-        /// dotnet run --project Examples.Console.csproj console
-        /// dotnet run --project Examples.Console.csproj inmemory
-        /// dotnet run --project Examples.Console.csproj zipkin -u http://localhost:9411/api/v2/spans
-        /// dotnet run --project Examples.Console.csproj jaeger -h localhost -p 6831
-        /// dotnet run --project Examples.Console.csproj prometheus -p 9464
-        /// dotnet run --project Examples.Console.csproj otlp -e "http://localhost:4317" -p "grpc"
-        /// dotnet run --project Examples.Console.csproj zpages
-        /// dotnet run --project Examples.Console.csproj metrics --help
+        /// dotnet run --project Examples.Console.csproj -- console
+        /// dotnet run --project Examples.Console.csproj -- inmemory
+        /// dotnet run --project Examples.Console.csproj -- zipkin -u http://localhost:9411/api/v2/spans
+        /// dotnet run --project Examples.Console.csproj -- jaeger -h localhost -p 6831
+        /// dotnet run --project Examples.Console.csproj -- prometheus -p 9464
+        /// dotnet run --project Examples.Console.csproj -- otlp -e "http://localhost:4317" -p "grpc"
+        /// dotnet run --project Examples.Console.csproj -- zpages
+        /// dotnet run --project Examples.Console.csproj -- metrics --help
+        ///
+        /// To see all available examples in the project run:
+        ///
+        /// dotnet run --project Examples.Console.csproj -- --help
         ///
         /// The above must be run from the project root folder
         /// (eg: C:\repos\opentelemetry-dotnet\examples\Console\).
-        /// </summary>
```

Comment:
intentional?

Reply:
Ops, letfover of trying to re-org the comment... fixing...